src/PumpStream.php

```diff
@@ -18,8 +18,8 @@
  */
 class PumpStream implements StreamInterface
 {
-    /** @var callable */
-    private $source;
+    /** @var callable|null */
+    private $source = null;
```

Comment:
changing the phpdoc is correct. but assigning null by default should be reverted. every class property is null by default.